src/core/retargeting_engine/python/deviceio_source_nodes/message_channel_sink.py

```
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Message channel sink node.
+
+Queues raw message payloads for delivery by MessageChannelSource.poll_tracker().
+"""
+
+from typing import TYPE_CHECKING, Deque
```

Ruff (0.15.7)

[warning] `typing.Deque` is deprecated, use `collections.deque` instead

(UP035)